lib/osf-components/addon/components/node-card/component.ts

```diff
@@ -95,6 +113,10 @@ export default class NodeCard extends Component {
         return false;
     }
 
+    get displayIcon(): boolean {
+        return !(this.node?.isRegistration || this.node?.isProject);
```

Comment:
This condition is always going to return `false`, since this component only accepts either Projects or Registrations. Is there a reason why this was added? What is this trying to do?

Reply:
In that case, I think we can just remove this logic and remove the icon-rendering piece in the template file

Reply:
Ok, looks like the icons can be removed!